dev/devicelab/bin/tasks/gradle_plugin_bundle_test.dart

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter_devicelab/framework/framework.dart';
+import 'package:flutter_devicelab/framework/apk_utils.dart';
+import 'package:flutter_devicelab/framework/utils.dart';
+
+String javaHome;
```

Comment:
Do these have to be globals? Can they at least go under main()?

Reply:
I cleaned this up, so no need for javaHome here :)